nilearn/regions/signal_extraction.py

```diff
@@ -76,6 +80,15 @@ def img_to_signals_labels(imgs, labels_img, mask_img=None,
     target_affine = imgs.affine
     target_shape = imgs.shape[:3]
 
+    # Check aggregating function
+    acceptable_funcs = ('mean', 'median',
```

Comment:
Rename `acceptable_funcs` to `valid_aggregation_functions`
Then remove the superfluous  comment `# Check aggregating function`

Reply:
Fixed